www/data/docs_sidebar.yml

```diff
@@ -1,122 +1,195 @@
 ---
 sidebar_links:
-- title: Getting Started
+- title:
   links:
-  - title: Overview
-    link: "/docs/overview/"
-  - title: Download and install Habitat
-    link: "/tutorials/download/"
-  - title: Tutorials
-    link: "/tutorials/"
-  - title: Concepts <span>&plus;</span>
-    link: "/docs/concepts-overview/"
+  - title: Docs Home
+    link: "/docs"
+  - title: Using Habitat <span>&plus;</span>
+    link: "/docs/using-habitat"
     sub_links:
+    - title: Overview
+      link: "#overview"
+    - title: Installing Habitat
+      link: "#install-habitat"
+    - title: Install FAQ
+      link: "#install-faq"
+    - title: Using Habitat Packages
+      link: "#using-packages"
+    - title: Service Groups
+      link: "#service-groups"
+    - title: Topologies
+      link: "#topologies"
+    - title: Config Updates
+      link: "#config-updates"
+    - title: File Uploads
+      link: "#file-uploads"
+    - title: Encryption
+      link: "#using-encryption"
+    - title: Update Strategies
+      link: "#using-updates"
+  - title: Developing Packages <span>&plus;</span>
+    link: "/docs/developing-packages"
+    sub_links:
+    - title: Writing a Plan.sh
+      link: "#write-plans"
+    - title: Config Templates
+      link: "#add-configuration"
+    - title: Binds and Exports
+      link: "#pkg-binds"
+    - title: Plan Builds
+      link: "#plan-builds"
+    - title: Troubleshooting Builds
+      link: "#debug-builds"
+    - title: Package Export Formats
+      link: "#pkg-exports"
+    - title: Share via Builder
+      link: "#sharing-pkgs"
+  - title: Commands (CLI) <span>&plus;</span>
+    link: "/docs/habitat-cli"
+    sub_links:
```

Comment:
The sidebar gets stuck on this because there are so many sublinks. Suggest only having top level CLI commands in the sidebar.

Reply:
fixed by new commit